roles/container-engine/cri-o/tasks/cleanup.yaml

```
@@ -118,3 +118,10 @@
     - cri-o
     - cri-o-runc
     - oci-systemd-hook
+
+- name: CRI-O | Remove CRI-O package configuration files
+  file:
+    name: "{{ item }}"
+    state: absent
+  loop:
+    - /etc/crio/crio.conf.d/01-crio-runc.conf
```

Why the loop instead of just using the filename directly ?

Alternatively, if you want to remove the whole conf dir :
```suggestion
    name: /etc/crio/crio.conf.d/
    state: absent
```

---

I've made changes to use the filename directly.